### PR TITLE
fix(router): ensure consistent request logs in error conditions

### PR DESCRIPTION
**Pull request by StarpTech**
## TODO

- [ ] Tests or benchmark included
- [ ] Documentation is changed or added on [https://app.gitbook.com/](https://app.gitbook.com/)
- [ ] PR title must follow [conventional-commit-standard](https://github.com/wundergraph/wundergraph/blob/main/CONTRIBUTING.md#conventional-commit-standard)

### CODE REVIEW

**Review by Slickstef11 — APPROVED** [review]

File: router/internal/recoveryhandler/recovery.go [hunk 1]
```diff
@@ -62,41 +33,6 @@ func (h *handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	defer func() {
 		if err := recover(); err != nil {
 			w.WriteHeader(http.StatusInternalServerError)
-
```
Comment (Aenimus) [inline router/internal/recoveryhandler/recovery.go:65]: This has all basically been moved into the logger defer?

**Review by Aenimus — APPROVED** [review]
LGTM (as far as I can tell)

File: router/internal/recoveryhandler/recovery.go (hunk 1 above)
Comment (StarpTech) [reply router/internal/recoveryhandler/recovery.go:65]: Yes, recovery only recovers and sets the status code.

**Review by df-wg — APPROVED** [review]
LGTM!

**Pull request merged**